po/si.po

```diff
@@ -0,0 +1,5815 @@
+# Sinhala translations for Geany package.
+# Copyright (C) 2022 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the Geany package.
+# aruna <aruna.hewapathirane@gmail.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Geany 1.2.4.1\n"
```

Comment:
The versionstring is a little weird -- but as it's not used much anyway no need to update it at the moment ;)